src/main/scala/zio/prelude/IdentityBoth.scala

```diff
@@ -71,7 +71,7 @@ object IdentityBoth extends LawfulF.Invariant[DeriveEqualIdentityBothInvariant,
       val any: Either[L, Any] = Right(())
 
       def both[A, B](fa: => Either[L, A], fb: => Either[L, B]): Either[L, (A, B)] =
-        fa.flatMap(a => fb.map(b => (a, b)))
+        fa.fold(Left(_), a => fb.map(b => (a, b)))
```

Comment:
Could define this as implicit syntax on Scala 2.11 like we do in ZIO so we don't have to use `fold` all the time.